systemtests/src/main/java/io/enmasse/systemtest/Environment.java

```diff
@@ -36,6 +37,8 @@
     public static final String TAG_ENV = "TAG";
     public static final String PRODUCT_NAME_ENV = "PRODUCT_NAME";
     public static final String APP_NAME_ENV = "APP_NAME";
+    public static final String INSTALL_TYPE = "INSTALL_TYPE";
+    public static final String OLM_INSTALL_TYPE = "OLM_INSTALL_TYPE";
```

Comment:
Is this redundant?

Reply:
I think it's not redundant, OLM_INSTALL_TYPE is a kind of subtype of OLM INSTALL_TYPE because as you know with OLM we can install in the default namespace (ie: openshift-operators) or in the namespace `enmasse-infra`